ivy/functional/backends/jax/layers.py

```diff
@@ -55,8 +55,8 @@ def _get_tranpose_padding(
 def conv1d(
     x: JaxArray,
     filters: JaxArray,
-    strides: Union[int, Tuple[int]],
-    padding: Union[str, int, Sequence[Tuple[int, int]]],
+    strides: Union[int, Tuple[int]] = 1,
```

Comment:
I think if we're changing the default value of strides for `conv1d` based on `torch.nn.functional.conv2d`, we should follow the same for the other convolution functions as well, as you've already done with the other backends of course!

Reply:
i thought the other convolution funcs were addressed in this pr did i miss any?

Reply:
Yep, just the `jax` backend ones, thanks!